plugin-animation/src/main/java/com/mapbox/maps/plugin/animation/CameraAnimatorsFactory.kt

```diff
@@ -102,7 +102,7 @@ class CameraAnimatorsFactory internal constructor(mapDelegateProvider: MapDelega
         animationList.add(
           CameraCenterAnimator(
             options = cameraAnimatorOptions(target) {
-              startValue(start)
+//              startValue(start)
```

Comment:
@kiryldz is this intended? I'm trying to port the changes in this PR to the gl-native bump